### ISSUE TITLE
Please provide installation instructions

### ISSUE THREAD

**Issue by visualcurrent** [pmendelski/google-addon-doc2md#2]
This looks like a great add-on! 
I don't see any installation instructions in the .md file.
Going through the Google Marketplace doesn't work because Google doesn't appear to like you. 

![image](https://user-images.githubusercontent.com/10997567/82746775-cce28b00-9dbd-11ea-9258-ce9d6f6ad9c7.png)

Is it possible to install from your github download?
Or is there a way to use your tool offline from a downloaded google doc?
Thanks for what looks like a cool tool for conversion!

**Comment by pmendelski** [comment]
Hi, thanks. For letting me know.
I've just sent few email and submitted dozen forms. Hopefully they will approve it ...sigh.

**Comment by visualcurrent** [comment]
I can only imagine how frustrating that would be. Thank you for creating it
and getting it approved again.

If it's not too much trouble, please ping me when it gets approved.

I'm checking out Obsidian.md and liking it so far. It will be nice to have
a smooth way to get my Google docs into there.

Best

**Comment by pmendelski** [comment]
Hi, Doc2Md has just got approved.
Please let me know in case of any installation problems.

Sorry, for taking so long to fix this.

**Comment by visualcurrent** [comment]
Congratulations!

Thanks for pinging me on this.

I've installed it and am excited to start using it.

The only weird thing that I encountered during installations was here

[image: image.png]

Clicking "Complete additional setup now" only takes me to your github page
and there's no additional information there about what additional setup is
required.

But it works! So no problem. Just thought that's worth pointing out.
